src/device/pf_cmina.c

```diff
@@ -293,6 +293,9 @@ int pf_cmina_set_default_cfg (pnet_t * net, uint16_t reset_mode)
 
          /* Reset I&M data */
          ret = pf_fspm_clear_im_data (net);
+
+         pf_file_clear (p_file_directory, PF_FILENAME_SYSCONTACT);
+         pf_file_clear (p_file_directory, PF_FILENAME_SYSNAME);
```

Comment:
The description for RESET_APPLICATION_DATA in PN-AL-Services ch. 6.3.11.3.2 does not state that sysContact or sysName should be reset. This is only stated in the description for RESET_COMMUNICATION_PARAMETER.

Reply:
Added comment in the code regarding additional requirement from test case specification.
Done.